### ISSUE TITLE
Modulizer Not Working

### ISSUE THREAD

**Issue by billstrahlend** [Falkeyn/Modulizer#9]
I am not sure what has happened, but Modulizer is no longer working for me. I have Sketch 39, the newest installment of Sketch.

**Comment by jbdesigns** [comment]
Same modulizer isn't working for me either Sketch 39.1

**Comment by Falkeyn** [comment]
Hi
I found what modulizer breaks at start on multiple selection (it should group it first, but catch the error and cant go on). In other cases it works normally. Do you have the same?

Try on this file
[test.sketch.zip](https://github.com/Falkeyn/Modulizer/files/413825/test.sketch.zip)

**Comment by billstrahlend** [comment]
You are correct. If you manually group the layers first, then select the group and do CMD+SHIFT+M it works fine. However, like you stated, if you have multiple layers selected, it does nothing.

**Comment by Falkeyn** [comment]
Thanks! I'll fix it soon
